bedrock/base/templates/includes/sticky-promo.html

```diff
@@ -0,0 +1,41 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% set promo_referrals = '?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=sticky-promo' %}
+{% if ftl_has_messages('firefox-sticky-promo-get-the-latest-firefox', 'firefox-sticky-promo-meet-our-family-of') %}
```

Comment:
@alexgibson Added the `ftl_has_messages` IF statement. I only tested for the two new strings, and the other strings are all brand/property names that should already be localized.